smirnoff_omm/smirnoff.ipynb

```diff
@@ -4,27 +4,35 @@
    "cell_type": "markdown",
```

Comment:
<pre class="ql-syntax" spellcheck="false">Line #13.    from simtk import openmm, unit
</pre><p>Should we update the import syntax for openmm here? </p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/mosdef-hub/mosdef-workflows/pull/35/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='2'/>

Reply:
Yes - but we need to make a new toolkit release for that to happen. I don't have an ETA on that, but we should probably not merge this until then.